### PR TITLE
utils:disk.py Modified the get_disk() method

### PR DESCRIPTION
**Pull request by shirishaganta1**
Added condition in order to get physical hardisks
and partitions

### CODE REVIEW

File: avocado/utils/disk.py [hunk 1]
```diff
@@ -118,10 +118,18 @@ def get_disks():
     :returns: a list of paths to the physical disks on the system
     :rtype: list of str
     """
-    json_result = process.run("lsblk --json --paths --inverse")
+    json_result = process.run('lsblk --json --paths --inverse')
     json_data = json.loads(json_result.stdout_text)
-    return [str(disk["name"]) for disk in json_data["blockdevices"]]
-
+    # Create an empty list to store the disk names
```
Comment (PraveenPenguin) [inline avocado/utils/disk.py:123]: please remove this comment and elsewhere as well

Comment (shirishaganta1) [reply avocado/utils/disk.py]: Done

**Comment by abdhaleegit** [comment]
@Naresh-ibm will this effect our block tests ?

**Comment by Naresh-ibm** [comment]
@shirishaganta1 please attach sample outputs.

**Comment by shirishaganta1** [comment]
> @shirishaganta1 please attach sample outputs.

Before Patch
# python disk.py 
['/dev/sda1', '/dev/sda2', '/dev/mapper/rhel_ltcever87--lp12-root', '/dev/mapper/rhel_ltcever87--lp12-swap', '/dev/mapper/rhel_ltcever87--lp12-home', '/dev/pmem0.3', '/dev/pmem0', '/dev/pmem0.2s']

After Patch
# python t.py
['/dev/sda1', '/dev/sda', '/dev/sda2', '/dev/sda', '/dev/mapper/rhel_ltcever87--lp12-root', '/dev/sda3', '/dev/mapper/rhel_ltcever87--lp12-swap', '/dev/sda3', '/dev/mapper/rhel_ltcever87--lp12-home', '/dev/sda3', '/dev/pmem0.3', '/dev/pmem0', '/dev/pmem0.2s']

**Comment by abdhaleegit** [comment]
@Naresh-ibm are we good with these changes

**Review by Naresh-ibm — APPROVED** [review]
LGTM

@shirishaganta1 Thanks for the PR

**Review by PraveenPenguin — APPROVED** [review]

**Comment by PraveenPenguin** [comment]
@shirishaganta1 PR good to go in can you please quickly resolve static check issue

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Review by clebergnu — CHANGES_REQUESTED** [review]
Hi @shirishaganta1,

This PR is failing a selftests:

```
[stdout] ======================================================================
selftests/unit/utils/disk.py:Disk.test_disks: FAIL
[stdout] FAIL: test_disks (disk.Disk)
[stdout] ----------------------------------------------------------------------
[stdout] Traceback (most recent call last):
[stdout]   File "selftests/unit/utils/disk.py", line 54, in test_disks
[stdout]     self.assertEqual(disk.get_disks(), ["/dev/vda"])
[stdout] AssertionError: Lists differ: ['/dev/vda', '/dev/vda1', '/dev/vda2', '/dev/vda3', '/dev/vda4'] != ['/dev/vda']
[stdout] 
[stdout] First list contains 4 additional elements.
[stdout] First extra element 1:
[stdout] '/dev/vda1'
[stdout] 
[stdout] - ['/dev/vda', '/dev/vda1', '/dev/vda2', '/dev/vda3', '/dev/vda4']
[stdout] + ['/dev/vda']
[stdout] 
[stdout] ----------------------------------------------------------------------
[stdout] Ran 1 test in 0.002s
[stdout] 
[stdout] FAILED (failures=1)
```

Can you please look into it?

Thanks!

**Comment by clebergnu** [comment]
Hi @shirishaganta1, did you have a chance to look into the test failure? Thanks!

**Comment by shirishaganta1** [comment]
> Hi @shirishaganta1, did you have a chance to look into the test failure? Thanks!

Hi @clebergnu 
No I haven't looked into the test failure.. Will try to spend some time and look into it..
Thanks

**Comment by richtja** [comment]
Hi @shirishaganta1, gentle ping here.

**Comment by shirishaganta1** [comment]
> Hi @shirishaganta1, gentle ping here.

Hi @richtja handled the test failure..

**Comment by richtja** [comment]
> > Hi @shirishaganta1, gentle ping here.
> 
> Hi @richtja handled the test failure..

Hi @shirishaganta1, thank you for quick reply. Can you please do the rebase to master, it should fix all the static check failures.

**Comment by shirishaganta1** [comment]
> > > Hi @shirishaganta1, gentle ping here.
> > 
> > 
> > Hi @richtja handled the test failure..
> 
> Hi @shirishaganta1, thank you for quick reply. Can you please do the rebase to master, it should fix all the static check failures.

Hi @richtja , yes done that now all checks are passed

**Review by PraveenPenguin — APPROVED** [review]
@shirishaganta1 thanks for patch

**Review by richtja — APPROVED** [review]
@shirishaganta1 thank you. It LGTM

**Pull request merged**